thorlcr/graph/thgraph.hpp

```diff
@@ -883,6 +883,7 @@ class graph_decl CJobBase : public CInterfaceOf<IInterface>, implements IExcepti
     memsize_t keyNodeCacheBytes = 0;
     memsize_t keyLeafCacheBytes = 0;
     memsize_t keyBlobCacheBytes = 0;
+    TraceFlags oldTraceFlags;
```

Comment:
```
TraceFlags oldTraceFlags = TraceFlags::None
```
?
I know gets initialize in an early method (but not ctor), but I am a fan of member initializing everything possible,